Libplanet/Blocks/Block.cs

```diff
@@ -68,7 +68,7 @@ public class Block<T> : IBlockExcerpt
             Address? miner,
             BlockHash? previousHash,
             DateTimeOffset timestamp,
-            IEnumerable<Transaction<T>> transactions,
+            IReadOnlyList<Transaction<T>> transactions,
```

Comment:
This public change needs to be recorded in the changelog file.

Reply:
Okay, I'll look into it.

Reply:
Done and done.